stores/sql/main.go

```diff
@@ -1824,17 +1826,17 @@ func RenameDirectories(ctx context.Context, tx sql.Tx, bucket, prefixOld, prefix
 	}
 	defer insertStmt.Close()
 
-	updateNameStmt, err := tx.Prepare(ctx, "UPDATE directories SET name = ? WHERE id = ?")
+	updateExistingStmt, err := tx.Prepare(ctx, "UPDATE directories SET name = ?, db_parent_id = ? WHERE id = ?")
```

Comment:
I overlooked dragging and dropping of files and folders. Only tested the `Rename` functionality.